shell/browser/api/electron_api_menu_mac.mm

```diff
@@ -54,6 +73,29 @@
   base::SequencedTaskRunnerHandle::Get()->PostTask(FROM_HERE, std::move(popup));
 }
 
+v8::Local<v8::Value> Menu::GetUserAcceleratorAt(int command_id) const {
+  v8::Isolate* isolate = JavascriptEnvironment::GetIsolate();
+  if (![NSMenuItem usesUserKeyEquivalents])
+    return v8::Null(isolate);
+
+  auto controller = base::scoped_nsobject<ElectronMenuController>(
+      [[ElectronMenuController alloc] initWithModel:model()
+                              useDefaultAccelerator:NO]);
+
+  base::scoped_nsobject<NSMenuItem> item =
+      [controller makeMenuItemForIndex:GetIndexOfCommandId(command_id)
+                             fromModel:model()];
```

Comment:
do we need to test for index !== -1 to ensure an index was found before using it?

Reply:
I think it's a theoretical impossible state for that method to return `-1` given this is a non-public binding and we control the command_id passinto this method.  I guess it's a reasonable safety check though